cupboard_app/views.py

```diff
@@ -769,3 +772,19 @@ def list(self, request: Request) -> Response:
         all_measurements = get_all_measurements()
         serializer = MeasurementSerializer(all_measurements, many=True)
         return Response(serializer.data)
+
+
+class UploadDataAPIView(APIView):
+    permission_classes = [AllowAny]
+
+    def get(self, request: Request) -> Response:
+        with open('temp.json', 'r', encoding="utf8") as file:
```

Comment:
For the file, we'll have it so that we have a folder structure like so
/src/food.json
This folder isn't part of the repository and devs can add any data files into this directory and run this script to add the ingredients.

Reply:
done 👍